infrastructure/ruby

```diff
@@ -1,27 +1,27 @@
-argo
-common-accessioning
-dlme
-dlme-transform
-dor-services-app
-dor_indexing_app
-dor-fetcher-service
-gis-robot-suite
-google-books
-hydra_etd
-hydrus
-lyberservices-scripts
-modsulator-app-rails
-pre-assembly
-preservation_catalog
-preservation_robots
-repository-api
-rialto-etl
-rialto-webapp
-robot-console
-sdr-services-app
-sul_pub
-suri_rails
-was_robot_suite
-was-thumbnail-service
-was-registrar-app
-workflow-server-rails
+sul-dlss/argo
+sul-dlss/common-accessioning
+sul-dlss/dlme
+sul-dlss/dlme-transform
+sul-dlss/dor-services-app
+sul-dlss/dor_indexing_app
+sul-dlss/dor-fetcher-service
+sul-dlss/gis-robot-suite
+sul-dlss/google-books
+sul-dlss/hydra_etd
+sul-dlss/hydrus
+sul-dlss/lyberservices-scripts
+sul-dlss/modsulator-app-rails
+sul-dlss/pre-assembly
+sul-dlss/preservation_catalog
+sul-dlss/preservation_robots
+sul-dlss/repository-api
+sul-dlss/rialto-etl
+sul-dlss/rialto-webapp
+sul-dlss/robot-console
+sul-dlss/sdr-services-app
+sul-dlss/sul_pub
+sul-dlss/suri_rails
+sul-dlss/was_robot_suite
+sul-dlss/was-thumbnail-service
+sul-dlss/was-registrar-app
+sul-dlss/workflow-server-rails
```

Comment:
did you want to add LD4P sinopia stuff in here?

Reply:
@ndushay I figured we could do that in a follow-up PR, but I'm open to it. cc: @justinlittman @jermnelson

Reply:
I assumed separate PR, but OK with either.